spec/requests/balance_pages_spec.rb

```diff
@@ -715,14 +716,23 @@
           create(:ach_account_stripe_succeed,
                  user: seller,
                  routing_number: "110000000",
-                 stripe_connect_account_id: "acct_1Qplf7S17V0i16U7",
-                 stripe_external_account_id: "ba_1Qplf7S17V0i16U7S5L4chWo",
-                 stripe_fingerprint: "dx7dqwoGHEQDKLLK",
+                 stripe_connect_account_id: "acct_12345",
+                 stripe_external_account_id: "ba_12345",
+                 stripe_fingerprint: "fingerprint",
                  )
-          create(:merchant_account, user: seller, charge_processor_merchant_id: "acct_1Qplf7S17V0i16U7")
+          create(:merchant_account, user: seller, charge_processor_merchant_id: "acct_12345")
 
           Credit.create_for_credit!(user: seller, amount_cents: 1000, crediting_user: seller)
           allow_any_instance_of(User).to receive(:compliant?).and_return(true)
+          allow_any_instance_of(User).to receive(:eligible_for_instant_payouts?).and_return(true)
```

Comment:
This is the only spec that didn't work with the new sandbox account as instant payouts are not supported on the new sandbox account yet. So mocked the Stripe API call here for now.